UPGRADE.md

```diff
@@ -9,6 +9,11 @@ The Doctrine\DBAL\Version class is no longer available: please refrain from chec
 1. The support of `PDO::PARAM_*`, `PDO::FETCH_*`, `PDO::CASE_*` and `PDO::PARAM_INPUT_OUTPUT` constants in the DBAL API is removed.
 2. `\Doctrine\DBAL\Driver\PDOStatement` does not extend `\PDOStatement` anymore.
 
+## BC BREAK: the SQLLogger interface has changed
```

Comment:
Finalizing implementors needs to be documented too.

Reply:
It's been done already in my [previous commit](https://github.com/doctrine/dbal/pull/3041/commits/6c5aacc91ab10571ce761f8895ff2fb07fa470cb).